src/wallet/rpcwallet.cpp

```diff
@@ -3323,16 +3323,20 @@ static UniValue bumpfee(const JSONRPCRequest& request)
         RPCTypeCheckObj(options,
             {
                 {"confTarget", UniValueType(UniValue::VNUM)},
+                {"conf_target", UniValueType(UniValue::VNUM)},
```

Comment:
Why is this adding a new key without documentation and whose value is silently ignored if `confTarget` is given? Doesn't make any sense to me and seems completely unrelated to this pull request anyway.

Reply:
Looks like this could be avoided entirely. But at least there should be a throw if both are set. You could also rename the documented options to `conf_target`, but keep `confTarget` as an undocumented alias for backwards compatibility. That should be in the release note though.

Reply:
Documentation updated/release notes added. Also throws if both are given.